mpp/build.gradle.kts

```diff
@@ -254,6 +263,44 @@ tasks.register("testComposeModules") { // used in https://github.com/JetBrains/a
     // android:exported needs to be explicitly specified for <activity>. Apps targeting Android 12 and higher are required to specify an explicit value for `android:exported` when the corresponding component has an intent filter defined.
 }
 
+tasks.register("jbApiDump") {
```

Comment:
Google added filtering logic around native binary validator for internal/experimental annotations. Can we re-use it in the future?

Reply:
We have to completely switch to another approach to reuse it. Created https://youtrack.jetbrains.com/issue/CMP-7845/Consider-using-AOSP-logic-for-API-checks